### PR TITLE
Add test for BUG # 1728: wrapping data creates long cells

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
https://github.com/UV-CDAT/uvcdat-testdata/pull/111

**Comment by danlipsa** [comment]
@aashish24 @doutriaux1 @sankhesh Please review.

**Comment by doutriaux1** [comment]
@danlipsa I only see an added test.

**Comment by danlipsa** [comment]
@doutriaux1 Indeed this is a test we wanted to add since we fixed this bug (adding +over to proj4).

**Comment by aashish24** [comment]
@danlipsa this is cool! so it works now? (I know I am asking obvious here :))

**Comment by doutriaux1** [comment]
I see! Ok then can you please clean up the test. I will annotate what's wrong with it.

File: testing/vcs/vcs_test_boxfill_robinson_wrap.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# proj4. See https://github.com/UV-CDAT/uvcdat/issues/1728 for more information.
+pth = os.path.join(os.path.dirname(__file__), "..")
+sys.path.append(pth)
+import checkimage
+
+
+cdmsfile = cdms2.open(os.path.join(vcs.sample_data, "clt.nc"))
+clt2 = cdmsfile('clt')
+clt3 = clt2(latitude=(-90.0, 90.0),squeeze=1,longitude=(-180, 200.0),time=('1979-01', '1988-12'),)
+axesOperations = eval("{'latitude': 'def', 'longitude': 'def', 'time': 'def'}")
```
Comment (doutriaux1) [inline testing/vcs/vcs_test_boxfill_robinson_wrap.py]: useless

File: testing/vcs/vcs_test_boxfill_robinson_wrap.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    if axesOperations[axis] == 'sum':
+        clt3 = cdutil.averager(clt3, axis='(%s)'%axis, weight='equal', action='sum')
+    elif axesOperations[axis] == 'avg':
+        clt3 = cdutil.averager(clt3, axis='(%s)'%axis, weight='equal')
+    elif axesOperations[axis] == 'wgt':
+        clt3 = cdutil.averager(clt3, axis='(%s)'%axis)
+    elif axesOperations[axis] == 'gtm':
+        clt3 = genutil.statistics.geometricmean(clt3, axis='(%s)'%axis)
+    elif axesOperations[axis] == 'std':
+        clt3 = genutil.statistics.std(clt3, axis='(%s)'%axis)
```
Comment (doutriaux1) [inline testing/vcs/vcs_test_boxfill_robinson_wrap.py]: the whole loop is useless

File: testing/vcs/vcs_test_boxfill_robinson_wrap.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+canvas = vcs.init()
+gmBoxfill = vcs.getboxfill('a_robinson_boxfill')
+args = []
+args.append(clt3)
+gmBoxfill.boxfill_type = 'linear'
```
Comment (doutriaux1) [inline testing/vcs/vcs_test_boxfill_robinson_wrap.py]: all these are unmodified, you can remove these `gmBoxfill` lines bellow

File: testing/vcs/vcs_test_boxfill_robinson_wrap.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        clt3 = cdutil.averager(clt3, axis='(%s)'%axis, weight='equal')
+    elif axesOperations[axis] == 'wgt':
+        clt3 = cdutil.averager(clt3, axis='(%s)'%axis)
+    elif axesOperations[axis] == 'gtm':
+        clt3 = genutil.statistics.geometricmean(clt3, axis='(%s)'%axis)
+    elif axesOperations[axis] == 'std':
+        clt3 = genutil.statistics.std(clt3, axis='(%s)'%axis)
+canvas = vcs.init()
+gmBoxfill = vcs.getboxfill('a_robinson_boxfill')
+args = []
```
Comment (doutriaux1) [inline testing/vcs/vcs_test_boxfill_robinson_wrap.py]: no need to use the `args`

File: testing/vcs/vcs_test_boxfill_robinson_wrap.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+gmBoxfill.yaxisconvert = 'linear'
+gmBoxfill.ymtics1 = ''
+gmBoxfill.ymtics2 = ''
+gmBoxfill.yticlabels1 = '*'
+gmBoxfill.yticlabels2 = '*'
+kwargs = {}
+kwargs[ 'cdmsfile' ] = cdmsfile.id
+kwargs['bg'] = 1
+args.append( gmBoxfill )
+canvas.plot( *args, **kwargs)
```
Comment (doutriaux1) [inline testing/vcs/vcs_test_boxfill_robinson_wrap.py]: `canvas.plot(clt3,gmBoxfill)` would do just a snice here

**Comment by danlipsa** [comment]
@aashish24 Yes, it did work since we added the +over option to proj4.

**Comment by danlipsa** [comment]
@doutriaux1 I cleaned up the test as you suggested.

**Comment by doutriaux1** [comment]
@danlipsa L#17-#47 can still be removed, all of the gmBoxfill attribute setting lines

**Comment by danlipsa** [comment]
@doutriaux1 Any more comments? I think this is ready to be merged.

**Pull request merged**